### DISCUSSION TITLE
How to start open source in github and how to get swags from github

### DISCUSSION THREAD

**Discussion by Ramsey99** [community/community#127593] [GitHub Education]
### Hi everyone,

I want to know how many ways I can use github except contributing to own account and hackathon outside github, and how to learn from github how to connect with innovative minds and how to get swags for free of cost. Is there any way I can earn from github?

**Comment by Thomas123477737** [comment]
How do you get robox

**Comment by Thomas123477737** [comment]
Eny won now eny robox codes

**Comment by Madhu-mac** [comment]
> ### Hi everyone,
> I want to know how many ways I can use github except contributing to own account and hackathon outside github, and how to learn from github how to connect with innovative minds and how to get swags for free of cost. Is there any way I can earn from github?

@Ramsey99 
Contribute to Open Source: Collaborate on projects, build your portfolio.
Join Communities: Engage in discussions, follow inspiring developers.

GitHub Discussions: Share knowledge, ask questions, collaborate.
Follow Influential Developers: Engage with their work.

You can also work on Educational Content: Create tutorials, courses, or blog posts to monetize.
